### ISSUE TITLE
Create a menu to update hosts file

### ISSUE THREAD

**Issue by crazy-max** [crazy-max/WindowsSpyBlocker#47]

**Comment by ltGuillaume** [comment]
Just a question on this: are the IP addresses these hostnames resolve to not included in the firewall rules?

**Comment by crazy-max** [comment]
Yes for some of them but others use a custom DNS policy, like load balancing with geo-location awareness

**Comment by asvc** [comment]
Is this still on the roadmap?

I am currently in the process of re-writing [stoptracking/windows10](https://github.com/stoptracking/windows10) and _"Update hosts with the telemetry domains"_ is a part of my TODO list.

I will happily integrate WindowsSpyBlocker as the part of repository if this can be added.
